tests/unit_tests/test_config.py

```diff
@@ -204,8 +204,8 @@ def temp_yaml_config_file(
             "env": {
                 "sources": [
                     {"kind": "dodal", "module": "dodal.adsim"},
-                    {"kind": "planFunctions", "module": "dls_bluesky_core.plans"},
-                    {"kind": "planFunctions", "module": "dls_bluesky_core.stubs"},
+                    {"kind": "planFunctions", "module": "dodal.plans"},
```

Comment:
I probably missed this in the last PR, but what happens if I just want to import some of the dodal plans, not all of them?

Reply:
In theory, not just for dodal.plans but for any plans module, you maintain a structure such that x.y.z is the plans you want imported, and x.y.a is plans that you don't want imported.
This is also why dodal.plan_stubs.wrapped on the line below.

Until we can tackle https://github.com/DiamondLightSource/blueapi/issues/681